lang/src/main/java/com/okta/commons/lang/Objects.java

```diff
@@ -661,7 +661,7 @@ public static String nullSafeToString(Object obj) {
             return nullSafeToString((short[]) obj);
         }
         String str = obj.toString();
-        return (str != null ? str : EMPTY_STRING);
+        return (Strings.hasText(str) ? str : EMPTY_STRING);
```

Comment:
Fixes below findbugs error:

Redundant nullcheck of str, which is known to be non-null in com.okta.commons.lang.Objects.nullSafeToString(Object) [com.okta.commons.lang.Objects] Redundant null check at Objects.java:[line 664] RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE